libc/test/src/compiler/stack_chk_guard_test.cpp

```diff
@@ -0,0 +1,26 @@
+//===-- Unittests for assert ----------------------------------------------===//
```

Comment:
nit: fix header

Reply:
done in 13df4945c04e4b145fd0e60e25020c2743537812